src/AppInstallerCLITests/YamlManifest.cpp

```diff
@@ -886,6 +888,198 @@ TEST_CASE("ValidateV1_6GoodManifestAndVerifyContents", "[ManifestValidation]")
     VerifyV1ManifestContent(mergedManifest, false, ManifestVer{ s_ManifestVersionV1_6 });
 }
 
+TEST_CASE("WriteV1SingletonManifestAndVerifyContents", "[ManifestCreation]")
+{
+    ManifestValidateOption validateOption;

```

Comment:
>ManifestValidateOption validateOption; [](http://example.com/codeflow?start=4&length=38)

nit: this is not needed if we are to use the defaults.

Same applies to all below new tests added

Reply:
removed validate option for these tests.